cocos/gfx/webgl2/webgl2-framebuffer.ts

```diff
@@ -90,6 +90,8 @@ export class WebGL2Framebuffer extends Framebuffer {
         };
 
```

Comment:
NB! In v3.8.0, the get height() method has an error in returning the width (line 82).